### PR TITLE
ENCD-4997-adjust-GM-schema-for-FCC-expts

### PR DESCRIPTION
**Pull request by iyoungworth**

### CODE REVIEW

File: src/encoded/schemas/genetic_modification.json [hunk 1]
```diff
@@ -70,14 +70,27 @@
                     ],
                     "properties": {
                         "purpose": {
-                                "enum": ["repression", "activation", "overexpression", "characterization", "screening"]
+                                "enum": ["repression", "characterization"]
```
Comment (yunhailuo) [inline src/encoded/schemas/genetic_modification.json]: Is it expression here?

**Comment by iyoungworth** [comment]
After fixing the code to make sure the upgrade happened (and included the edge case), errors cropped up with previous tests on the GM object. It looked like some GM objects were then valid for multiple sections in a 'oneOf' clause, so I've tried to make my additions in a different way.
